torchchat/cli/convert_hf_checkpoint.py

```diff
@@ -41,7 +42,12 @@ def convert_hf_checkpoint(
     print(f"Model config {config.__dict__}")
 
     # Load the json file containing weight mapping
-    model_map_json = model_dir / "pytorch_model.bin.index.json"
+    model_map_json_matches = [Path(m) for m in glob.glob(str(model_dir / "*.index.json"))]
+    assert len(model_map_json_matches) <= 1, "Found multiple weight mapping files"
```

Comment:
Why is this an error? Thanks!

Reply:
Good catch. Seem my response over on your PR: https://github.com/pytorch/torchchat/pull/1346#issuecomment-2460257435